### PR TITLE
AUTH-251: command line override parameter names updated

### PR DESCRIPTION
**Pull request by kconkas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,9 +113,9 @@ Using the following command only the Identity Service and the [nginx-ingress](ht
 helm repo add alfresco-stable https://kubernetes-charts.alfresco.com/stable
 
 helm install alfresco-stable/alfresco-infrastructure \
-  --set alfresco-infrastructure.activemq.enabled=false \
-  --set alfresco-infrastructure.nginx-ingress.enabled=true \
-  --set alfresco-infrastructure.alfresco-identity-service.enabled=true \
+  --set activemq.enabled=false \
+  --set nginx-ingress.enabled=true \
+  --set alfresco-identity-service.enabled=true \
```
Comment (svidrascu) [inline README.md]: This should be in accordance with the conditions set on the infrastructure chart. https://github.com/Alfresco/alfresco-infrastructure-deployment/blob/master/helm/alfresco-infrastructure/requirements.yaml

File: README.md [hunk 2]
```diff
@@ -261,10 +261,10 @@ kubectl create secret generic realm-secret \
...
 
 helm install alfresco-stable/alfresco-infrastructure \
-  --set alfresco-infrastructure.activemq.enabled=false \
-  --set alfresco-infrastructure.nginx-ingress.enabled=true \
-  --set alfresco-infrastructure.alfresco-identity-service.enabled=true \
-  --set alfresco-infrastructure.alfresco-identity-service.keycloak.keycloak.extraArgs="-Dkeycloak.import=/realm/realm.json" \
+  --set activemq.enabled=false \
+  --set nginx-ingress.enabled=true \
+  --set alfresco-identity-service.enabled=true \
+  --set alfresco-identity-service.keycloak.keycloak.extraArgs="-Dkeycloak.import=/realm/realm.json" \
```
Comment (svidrascu) [inline README.md]: This line is correct

File: README.md [hunk 3]
```diff
@@ -216,7 +216,7 @@ For added resilience, we rely on support in the Keycloak chart for specifying mu
 
 ```bash
 
-  --set alfresco-infrastructure.alfresco-identity-service.keycloak.keycloak.replicas=3
+  --set alfresco-identity-service.keycloak.keycloak.replicas=3
```
Comment (svidrascu) [inline README.md]: This line is correct

File: README.md (hunk 1 above)
Comment (kconkas) [reply README.md]: Fixed all values to follow the `requirements.yaml` conditions.

**Pull request merged**